### PR TITLE
Velocity Support

### PR DESCRIPTION
**Pull request by Lorias-Jak**
added velocity support

will test it tomorrow on my live server. Didn't had the time today

### CODE REVIEW

**Comment by Lorias-Jak** [comment]
Works fine. Ready for merge

File: velocity/src/main/java/me/confuser/banmanager/webenhancer/velocity/VelocityPlugin.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                        new VelocityCommand(cmd, this);
+                }
+
+                logger.info("Registered commands");
+        }
+
+        public void setupRunnables() {
+//    Runner syncRunner = new Runner(new ExpiresSync(plugin));
+
+//    getServer().getScheduler().runTaskTimerAsynchronously(this, syncRunner, 10L, 10L);
```
Comment (confuser) [inline velocity/src/main/java/me/confuser/banmanager/webenhancer/velocity/VelocityPlugin.java:151]: Should this not be uncommented?

File: bungee/src/main/java/me/confuser/banmanager/webenhancer/bungee/BungeePlugin.java [hunk 2]
```diff
@@ -97,7 +97,7 @@ private PluginInfo setupConfigs() throws IOException {
 
   private void setupCommands() {
     for (CommonCommand cmd : plugin.getCommands()) {
-      new BungeeCommand(cmd, this);
+      new me.confuser.banmanager.webenhancer.bungee.BungeeCommand(cmd, this);
```
Comment (confuser) [inline bungee/src/main/java/me/confuser/banmanager/webenhancer/bungee/BungeePlugin.java]: Is this change needed?

File: velocity/src/main/java/me/confuser/banmanager/webenhancer/velocity/VelocityPlugin.java (hunk 1 above)
Comment (Lorias-Jak) [reply velocity/src/main/java/me/confuser/banmanager/webenhancer/velocity/VelocityPlugin.java:151]: In this case, I only copied the bungee and bukkit code. It's also commented out there, even if I didn't quite understand why. Made the duplicate in case you want to work on this, but I can delete it if you want to,

File: bungee/src/main/java/me/confuser/banmanager/webenhancer/bungee/BungeePlugin.java (hunk 2 above)
Comment (Lorias-Jak) [reply bungee/src/main/java/me/confuser/banmanager/webenhancer/bungee/BungeePlugin.java]: Oh, no thats my fault. Did an oopsie there

**Review by confuser — APPROVED** [review]
